cmake/SearchForStuff.cmake

```
@@ -233,3 +233,12 @@ if((GCC_VERSION VERSION_EQUAL "9.0" OR GCC_VERSION VERSION_GREATER "9.0") AND GC
     Even with that patch, compiling with LTO may still segfault. Use at your own risk!
     This text being in a compile log in an open issue may cause it to be closed.")
 endif()
+
+find_package(fmt "7.0.3")
```

```suggestion
find_package(fmt "6.2.1")
```
I am running this on NixOS and that is the fmt version it comes with, I successfully built pcsx2 and booted into the bios.
I don't know much about the CMake syntax, so I don't know if this also allows using higher versions.

---

>  don't know much about the CMake syntax, so I don't know if this also allows using higher versions.

It allows higher versions so your suggestion is right. However @xTVaser requested "atleast `v7`" earlier.

---

I guess it would be best to just stick to the recommended version to prevent issues down the line, even though this older one seems to work.